cfpb/cfgov-refresh
# Removing sheer.js and other dead js routes

Removing sheer.js and other dead js routes


## Removals

- Removed sheer.js and other routes, which are no longer needed after the conversion to Wagtail.

## Testing

1. Run gulp clean && gulp build.
2. Visit `http://localhost:8000/about-us/the-bureau/bureau-structure/`  and verify the bureau-structure chart works as expected. 

## Screenshots


## Checklist
